python/hsml/constants.py

```diff
@@ -25,6 +25,10 @@ class MODEL:
     FRAMEWORK_SKLEARN = "SKLEARN"
 
 
+class MODEL_REGISTRY:
+    HOPSFS_MOUNT_PREFIX = "/home/yarnapp/hopsfs/"
```

Comment:
Do you know that ~/Projects is also mounted inside the user's private directory in the container?
That way you might avoid the need for this prefix?

Reply:
Set keep_original_files to false by default.

Reply:
I didn't know. But we are using the absolute path internally. If a relative path is provided, we prepend os.getcwd() which will return an absolute path. So we need this prefix to double-check if it's a path in hopsfs-mount.